src/browser/app/profile/zen-browser.js

```diff
@@ -162,3 +162,6 @@ pref("sidebar.verticalTabs", false, locked);
 // Better Windows theming
 pref("widget.non-native-theme.scrollbar.style", 2);
 pref("widget.non-native-theme.use-theme-accent", true);
+
+// Closing the last tab should not cause the browser to close, this is bad default inherited from Firefox
```

Comment:
Make default screen with todo lists, tasks and calendar including desktop icons. Like a landing page after closing last tab